openedx/core/lib/block_structure/transformers.py

```diff
@@ -61,7 +62,11 @@ def __iadd__(self, transformers):
                 "The following requested transformers are not registered: {}".format(unregistered_transformers)
             )
 
-        self._transformers.extend(transformers)
+        for transformer in transformers:
```

Comment:
Thanks! I thought the comparison became a lot cleaner with the new subclassing/`isinstance` model of specifying transformer types.